extension/lib/main.js

```diff
@@ -6,11 +6,11 @@
 
```

Comment:
Would be nice to have but I would say lets push this out to the next release if possible. If we can have it ready before end of month we could fold those tests into the 0.7 release too.